timemachine/potentials/bonded.py

```diff
@@ -175,23 +175,23 @@ def harmonic_angle(conf, params, box, lamb, angle_idxs, lamb_mult=None, lamb_off
     lamb: float
         alchemical lambda parameter, linearly rescaled
 
+    angle_idxs: shape [num_angles, 3] np.array
```

Comment:
super tiny nit:
```suggestion
    angle_idxs: shape [num_angles, 3] np.ndarray
```
(`np.array` is a function)

Reply:
Applied this change to all instances of `"np.array"` as type in `timemachine/potentials` docstrings in https://github.com/proteneer/timemachine/pull/842/commits/200ebce9d8faa508b2cca46aecfe639d9f0c5cda